components/scream/src/physics/p3/scream_p3_interface.F90

```diff
@@ -64,6 +64,32 @@ subroutine p3_init_f90 (q,T,zi,pmid,pdel,ast,naai,npccn) bind(c)
     integer(kind=c_int) :: i, k
     logical(kind=c_bool) :: masterproc
 
+    call p3_init(micro_p3_lookup_dir,micro_p3_tableversion)
```

Comment:
Looks like p3_init_f90 is no longer using any of its inputs. If so, we should change its signature (avoid unused stuff).

Reply:
oh yeah, for sure.  I'll make the change and test.